compiler/installer.ini

```diff
@@ -4,6 +4,8 @@
 [Project]
 Name: "Nim"
 Version: "$version"
+Commit: "$commit"
```

Comment:
worth adding repository here?

Reply:
Change the author below?

Reply:
I'm just patching things here and there to get stuff going. In the future I'd get rid of this file and niminst both. Hence I try not to edit too much.

Reply:
An another approach would be to turn these into special niminst parameters instead of this weird "fields in a config" approach, if you think that one would be better.